Import a list of domains to the Allowlist/Denylist

manelpb opened the pull request.

manelpb committed:
Import a list of domains to the Allowlist/Denylist

manelpb committed:
fix formatting

manelpb committed:
fix

hjk789 commented:
Thanks for the contribution! 

I have some things to point:

- Your changes in the Readme removed the trailing spaces, which were required to break the lines. As you can see in your fork's homepage, the formatting got messed up. Please add them back.
- In the script, in line 1388, you changed the condition to how it was in previous versions, which would cause a regression in the waitForLists interval. Please change it back to the current condition, which is `if (document.querySelectorAll(".list-group-item").length > 1)`
- Did you tested the "Add" button? It reloads the page when clicked before even making the request, because of the form element. You have to take out the elements from inside it and remove the form, like it was done with the search bar in the Logs page.
- Visually it's good. I never thought of changing the input to a textarea. Nice!

manelpb committed:
CR

manelpb commented:
- Sorry about the readme lol I think i fixed now.
- I added back the condition. 
- Yeah, I did with page reload just to not spend too much time trying to add the elements in the list. I was inspecting the code and they built that with react, binding the onclick to buttons, etc. So I though the easier way for now was to just reload the page. I tested mine and didn't have issues with reloading before the elements were added, but maybe my Internet is just too fast lol

hjk789 commented:
Well, my connection is also kinda fast, but after multiple tests it still reloaded the page before even making the request. I suggest removing the form and replacing it with the textarea.
